packages/block-library/src/footnotes/index.php

```diff
@@ -68,6 +68,22 @@ function render_block_core_footnotes( $attributes, $content, $block ) {
  * @since 6.3.0
  */
 function register_block_core_footnotes() {
+	register_block_type_from_metadata(
+		__DIR__ . '/footnotes',
+		array(
+			'render_callback' => 'render_block_core_footnotes',
+		)
+	);
+}
+add_action( 'init', 'register_block_core_footnotes' );
+
+
+/**
+ * Registers the footnotes meta field required for footnotes to work.
+ *
+ * @since 6.5.0
+ */
+function wp_register_footnotes_post_meta() {
```

Comment:
Maybe??

```suggestion
function register_block_core_footnotes_post_meta() {
```

I'm unclear why the post types are limited to public. If someone has a private post type using the block editor, they probably want access to footnotes.

That said, if there are reasons then post types can be `public` but not actually public. You'll need to change the code below to:

```php
	$post_types = get_post_types( array( 'show_in_rest' => true ) );
	$post_types = array_filter( $post_types, 'is_post_type_viewable' ) );
```

Outside the diff sorry so can't do an inline suggestion

Reply:
Hi @peterwilsoncc, your points are well taken! Footnotes work even on non-public post types, so I removed the public filter. I also updated the function name. Thank you for the suggestions 👍